DevelopmentProposals/template.md

```diff
@@ -0,0 +1,61 @@
+## Project: TITLE
+***OpenSim Design Document***
+
+### Overview
+
+Just a few sentences that describes the purpose of this project. 
+For example:
+*Add ability to read/write table of quaternions to/from STO files.*
+
+### Requirements
+
+List the set of requirements that this project must fulfill. Include all constraints this proposal is working within.
+If the list gets too long, consider splitting the project into multiple small projects.
+
+For example:
+* Add support for reading and writing of `TimeSeriesTables` of `Quaternion`s to/from STO files through the existing class `STOFileAdapter`.
+
+### Architecture
+Describe changes to the architecture and include a diagram if applicable/helpful.
+This should be a conceptual diagram that describes what components of OpenSim will be utilized or changed, the flow of information, new classes, etc.
+
+For example:
+Existing classes `STOFileAdapter` and `DelimFileAdapter` will be edited by adding template specializations. The existing dispatch mechanism will be extended to include `Quaternion`. ....
+
+### Interfaces
+Describe any new interfaces or modifications to interfaces, function API changes, SDF changes. These changes can be notional. Describe if/how/when OpenSim users/developers will be affected.
+
+For example:
+There will no change to the existing interface. This proposal is to add ability to read and write quaternion tables to files using existing interface.
+
+#### Bindings
+Describe implications to Python and Java bindings.
```

Comment:
Very minor wording "implications to" -> "implications for". Same change on line 35.